source/_integrations/landisgyr_heat_meter.markdown

```diff
@@ -0,0 +1,88 @@
+---
+title: Landis+Gyr Heat Meter
+description: Instructions on how to integrate your Landis+Gyr Heat Meter device into Home Assistant.
+ha_category:
+  - Energy
+  - Sensor
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2022.7
+ha_domain: landisgyr_heat_meter
+ha_codeowners:
+  - '@vpathuis'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Landis+Gyr Heat Meter integration for Home Assistant allows you to read the meter values from Ultraheat devices by Landis & Gyr. These devices are commonly used for district heating. The integration has been tested with the Landis & Gyr Ultraheat 50 (UH50) type LUGCUH50. Other models are likely to work as well.
+
+The device is read through the optical interface. An (USB) IR reader is needed and connected to Home Assistant.
+
+<img src='/images/integrations/landisgyr_heat_meter/usb_ir_reader.png' />
+
+<div class='note warning'>
```

Comment:
Is this really a warning? This could also just be a normal text at the end of the sensors paragraph.